### ISSUE TITLE
stuck in second step of "Preprocessing with the Tagger"

### ISSUE THREAD

**Issue by wormsleep** [dsindex/syntaxnet#12]
Thank you very much for the tutorial. But I have the following problems with the Mac OSX operating system, I stuck in second step of "Preprocessing with the Tagger". Which in your train.sh is preprocess_with_tagger step.

I have completed the first step that is "Training the SyntaxNet POS Tagger", and generated directory models, 

My directory structure is as follows:

--models/syntaxnet
-- english/ # Download from Universal Dependencies and use UD_English
---- en-ud-dev.conllu
---- en-ud-test.conllu
---- en-ud-train.conllu
-- trainer.sh # Training the SyntaxNet POS Tagger script
-- preprocessingWithTagger.sh
-- syntaxnet/context.pbtxt # modified params of the file_pattern flag training-corpus, tuning-corpus, dev-corpus correspond to english/en-ud-train.conllu.conv, english/en-ud-test.conllu.conv, english/en-ud-dev.conllu.conv

The script about trainer.sh:

```
#!/bin/bash

bazel-bin/syntaxnet/parser_trainer \
      --task_context=syntaxnet/context.pbtxt \
      --arg_prefix=brain_pos \
      --compute_lexicon \
      --graph_builder=greedy \
      --training_corpus=training-corpus \
      --tuning_corpus=tuning-corpus \
      --output_path=models \
      --batch_size=256 \
      --decay_steps=3600 \
      --hidden_layer_sizes=128 \
      --learning_rate=0.08 \
      --momentum=0.9 \
      --seed=0 \
      --params=128-0.08-3600-0.9-0
```

This script looks normal when it runs, and it generates a directory named models:
models/
models//brain_pos
models//brain_pos/greedy
models//brain_pos/greedy/128-0.08-3600-0.9-0
models//brain_pos/greedy/128-0.08-3600-0.9-0/category-map
models//brain_pos/greedy/128-0.08-3600-0.9-0/char-map
models//brain_pos/greedy/128-0.08-3600-0.9-0/checkpoint
models//brain_pos/greedy/128-0.08-3600-0.9-0/context
models//brain_pos/greedy/128-0.08-3600-0.9-0/graph
models//brain_pos/greedy/128-0.08-3600-0.9-0/label-map
models//brain_pos/greedy/128-0.08-3600-0.9-0/latest-model
models//brain_pos/greedy/128-0.08-3600-0.9-0/latest-model.meta
models//brain_pos/greedy/128-0.08-3600-0.9-0/lcword-map
models//brain_pos/greedy/128-0.08-3600-0.9-0/model
models//brain_pos/greedy/128-0.08-3600-0.9-0/model.meta
models//brain_pos/greedy/128-0.08-3600-0.9-0/prefix-table
models//brain_pos/greedy/128-0.08-3600-0.9-0/status
models//brain_pos/greedy/128-0.08-3600-0.9-0/suffix-table
models//brain_pos/greedy/128-0.08-3600-0.9-0/tag-map
models//brain_pos/greedy/128-0.08-3600-0.9-0/tag-to-category
models//brain_pos/greedy/128-0.08-3600-0.9-0/word-map

next I run the script preprocessingWithTagger.sh, 

```
#!/bin/bash

PARAMS=128-0.08-3600-0.9-0
for SET in training tuning dev; do
  bazel-bin/syntaxnet/parser_eval \
    --task_context=models/brain_pos/greedy/$PARAMS/context \
    --hidden_layer_sizes=128 \
    --input=$SET-corpus \
    --output=tagged-$SET-corpus \
    --arg_prefix=brain_pos \
    --graph_builder=greedy \
    --model_path=models/brain_pos/greedy/$PARAMS/model
done
```

console show the above error:
(tensorflow) zhaoweideMacBook-Pro:syntaxnet zhaowei$ ./preprocessingWithTagger.sh

Part of error code below:

Caused by op u'save/Assign_15', defined at:
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/main/syntaxnet/parser_eval.py", line 161, in 
tf.app.run()
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/org_tensorflow/tensorflow/python/platform/app.py", line 30, in run
sys.exit(main(sys.argv[:1] + flags_passthrough))
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/main/syntaxnet/parser_eval.py", line 157, in main
Eval(sess)
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/main/syntaxnet/parser_eval.py", line 113, in Eval
parser.AddSaver(FLAGS.slim_model)
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/main/syntaxnet/graph_builder.py", line 568, in AddSaver
self.saver = tf.train.Saver(variables_to_save)
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/org_tensorflow/tensorflow/python/training/saver.py", line 1078, in init
self.build()
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/org_tensorflow/tensorflow/python/training/saver.py", line 1107, in build
restore_sequentially=self.restore_sequentially)
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/org_tensorflow/tensorflow/python/training/saver.py", line 705, in build
restore_sequentially, reshape)
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/org_tensorflow/tensorflow/python/training/saver.py", line 454, in AddRestoreOps
assign_ops.append(saveable.restore(tensors, shapes))
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/org_tensorflow/tensorflow/python/training/saver.py", line 211, in restore
self.op.get_shape().is_fully_defined())
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/org_tensorflow/tensorflow/python/ops/gen_state_ops.py", line 45, in assign
use_locking=use_locking, name=name)
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/org_tensorflow/tensorflow/python/framework/op_def_library.py", line 749, in apply_op
op_def=op_def)
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/org_tensorflow/tensorflow/python/framework/ops.py", line 2386, in create_op
original_op=self._default_original_op, op_def=op_def)
File "/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_eval.runfiles/org_tensorflow/tensorflow/python/framework/ops.py", line 1298, in __init
self._traceback = _extract_stack()

InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [32,50] rhs shape= [256,50]
[[Node: save/Assign_15 = AssignT=DT_FLOAT, _class=["loc:@transition_scores"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/cpu:0"]]
